spec/filters/uuid_spec.rb

```diff
@@ -2,4 +2,48 @@
 require 'logstash/filters/uuid'
 
 describe LogStash::Filters::Uuid do
+  describe "add field" do
+    config <<-CONFIG
+      filter {
+        uuid {
+          target => "uuid"
+        }
+      }
+    CONFIG
+
+    sample("foo" => "bar") do
```

Comment:
We try to move away from this kind of specs, to an style more similar to what you can see at #3